examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -33,6 +33,20 @@ using namespace ::chip::Controller;
 CHIP_ERROR PairingCommand::RunCommand()
 {
     CurrentCommissioner().RegisterPairingDelegate(this);
+    if (mCASEAuthTags.HasValue() && mCASEAuthTags.Value().size() <= kMaxSubjectCATAttributeCount)
+    {
+        CATValues cats = kUndefinedCATs;
+        for (size_t index = 0; index < mCASEAuthTags.Value().size(); ++index)
+        {
+            cats.values[index] = mCASEAuthTags.Value()[index];
+        }
+        if (cats.AreValid())
+        {
+            ExampleOperationalCredentialsIssuer * opIssuer =
+                reinterpret_cast<chip::Controller::ExampleOperationalCredentialsIssuer *>(mCredIssuerCmds->GetCredentialIssuer());
+            opIssuer->SetCATValuesForNextNOCRequest(cats);
+        }
+    }
```

Comment:
We should `SetCATValuesForNextNOCRequest` with empty cats up front in this method.  Otherwise in interactive mode if a pairing command is run with CATs and another one is run without CATs the second one will end up using the same CATs as the first one, as far as I can tell.

Reply:
Done